.github/workflows/build-citus-community-nightlies.yml

```diff
@@ -1,7 +1,7 @@
 name: Build and publish citus community nightly packages
 
 env:
-  MAIN_BRANCH: "all-citus"
+  MAIN_BRANCH: "all-citus-pg15-beta"
```

Comment:
Do we intend to keep this change, or is it temporary?

Reply:
This is temporary. I will not merge the PR for all-citus-pg15-beta either. (https://github.com/citusdata/packaging/pull/956)